ext/curl/interface.c

```diff
@@ -1731,6 +1731,7 @@ static zend_result _php_curl_setopt(php_curl *ch, zend_long option, zval *zvalue
 		case CURLOPT_HTTPPROXYTUNNEL:
 		case CURLOPT_HTTP_VERSION:
 		case CURLOPT_INFILESIZE:
+		case CURLOPT_INFILESIZE_LARGE:
```

Comment:
nit: I suggest to put it after `CURLOPT_TIMEVALUE_LARGE` instead.

Reply:
fixed, makes sense